rosa_tutorials/rosa-consistent-egress-external-traffic.adoc

```diff
@@ -0,0 +1,419 @@
+  #       are deployed.
+  loadBalancerSourceRanges:
+    - 10.10.100.254/32
+    - 10.10.150.254/32
+    - 10.10.200.254/32
+    - 10.10.100.253/32
+    - 10.10.150.253/32
+    - 10.10.200.253/32
+EOF
+----
+
+Retrieve the load balancer hostname as the `LOAD_BALANCER_HOSTNAME` environment variable which you can copy and use for following steps:
+
+[source,terminal]
+----
+$ export LOAD_BALANCER_HOSTNAME=$(oc get svc -n default demo-service -o json | jq -r '.status.loadBalancer.ingress[].hostname')
+----
+
+==== Test namespace egress
+
+Test the namespace egress rule which was created previously. The following starts an interactive shell which allows you to run curl against the demo service:
+
+[source,terminal]
+----
+$ oc run \
+  demo-egress-ns \
+  -it \
+  --namespace=demo-egress-ns \
+  --env=LOAD_BALANCER_HOSTNAME=$LOAD_BALANCER_HOSTNAME \
+  --image=registry.access.redhat.com/ubi9/ubi -- \
+  bash
+----
+
+Once inside the pod, you can send a request to the load balancer, ensuring that you can successfully connect:
+
+[source,terminal]
+----
+$ curl -s http://$LOAD_BALANCER_HOSTNAME
+----
+
+You should see output similar to the following, indicating a successful connection. It should be noted that that `client_address` below is the internal IP address of the load balancer rather than our egress IP. Successful connectivity (by limiting the service to `.spec.loadBalancerSourceRanges`) is what provides a successful demonstration:
+
+.Example output
+[source,terminal]
+----
+CLIENT VALUES:
+client_address=10.10.207.247
+command=GET
+real path=/
+query=nil
+request_version=1.1
+request_uri=http://internal-a3e61de18bfca4a53a94a208752b7263-148284314.us-east-1.elb.amazonaws.com:8080/
+
+SERVER VALUES:
+server_version=nginx: 1.10.0 - lua: 10001
+
+HEADERS RECEIVED:
+accept=*/*
+host=internal-a3e61de18bfca4a53a94a208752b7263-148284314.us-east-1.elb.amazonaws.com
+user-agent=curl/7.76.1
+BODY:
+-no body in request-
+----
+
+You can safely exit the pod once you are done:
+
+[source,terminal]
+----
+$ exit
+----
+
+==== Test pod egress
+
+Test the pod egress rule which was created previously. The following starts an interactive shell which allows you to run curl against the demo service:
+
+[source,terminal]
+----
+$ oc run \
+  demo-egress-pod \
+  -it \
+  --namespace=demo-egress-pod \
+  --env=LOAD_BALANCER_HOSTNAME=$LOAD_BALANCER_HOSTNAME \
+  --image=registry.access.redhat.com/ubi9/ubi -- \
+  bash
+----
+
+Once inside the pod, you can send a request to the load balancer, ensuring that you can successfully connect:
+
+[source,terminal]
+----
+$ curl -s http://$LOAD_BALANCER_HOSTNAME
+----
+
+You should see output similar to the following, indicating a successful connection.  It should be noted that that `client_address` below is the internal IP address of the load balancer rather than our egress IP.  Successful connectivity (by limiting the service to `.spec.loadBalancerSourceRanges`) is what provides a successful demonstration:
```

Comment:
```suggestion
You should see output similar to the following, indicating a successful connection. It should be noted that the `client_address` below is the internal IP address of the load balancer rather than our egress IP.  Successful connectivity (by limiting the service to `.spec.loadBalancerSourceRanges`) is what provides a successful demonstration:
```